pkg/scheduler/scheduling_application.go

```diff
@@ -598,7 +606,7 @@ func (sa *SchedulingApplication) tryNodes(ask *schedulingAllocationAsk, nodeIter
 		// nothing allocated should we look at a reservation?
 		// TODO make this smarter a hardcoded delay is not the right thing
 		askAge := time.Since(ask.getCreateTime())
-		if allowReserve && askAge > reservationDelay {
+		if !reservationDisabled && allowReserve && askAge > reservationDelay {
```

Comment:
Minor thing: could we move the `reservationDisabled ` check to the former line in this function:
```
allowReserve := len(reservedAsks) < int(ask.pendingRepeatAsk)
```